### ISSUE TITLE
I can't find language change feature. 

### ISSUE THREAD

**Issue by pavelsust** [OxygenCobalt/Auxio#64]
Can you please add language change feature into this application.

**Comment by OxygenCobalt** [comment]
Can't you just change the system language?

**Comment by pavelsust** [comment]
> That seems redundant. Can't one just change the system language?

You can add language change from the application.

**Comment by OxygenCobalt** [comment]
Is there a benefit for application-specific language configuration?

**Comment by vladimirkosolapov** [comment]
«Panlingual» feature for per-app language settings planned for Android 13, so there is no point in adding this option to the app now.

**Comment by OxygenCobalt** [comment]
Sure, but version fragmentation fragmentation ensures that many people will not be on Android 13 for awhile. From what I've seen, a lot of non-native english speakers keep the system language as English because the translations are poor, but still enable their native language on an app-by-app basis. There might be some use in this.

**Comment by OxygenCobalt** [comment]
Judging by the new [1.6.0-alpha01 AppCompat Release](https://developer.android.com/about/versions/13/features/app-languages#androidx-impl), I can add this even sooner actually. Still no ETA, but it seems like an easy feature to add.

**Comment by OxygenCobalt** [comment]
Android 13 has hit platform stability, so I'm going to start working on this, and it should arrive in 2.5.0. However, I'm not sure whether the notification (or similar) will follow the in-app language or the system language, mostly due to possible technical complexity.

**Comment by FEER0** [comment]
Is this still planned for a future update?

**Comment by OxygenCobalt** [comment]
Yes @FEER0, it's just very complicated since Auxio has to refresh the strings it exposes to the android system itself (like the `MediaSession`)
